src/Aspire.Dashboard/Components/Controls/ResourceSelectOptionTemplate.razor

```diff
@@ -15,8 +15,7 @@
     }
     else if (ViewModel.Id?.Type is OtlpApplicationType.Instance)
     {
-        <div style="width: 20px;"></div>
-        <span aria-label="@string.Format(@Loc[nameof(ControlsStrings.ResourceDropdownReplicaAccessibleTitle)], ViewModel.Name, ViewModel.Id.ReplicaSetName)">@ViewModel.Name</span>
+        <span style="padding-left: 20px;" aria-label="@string.Format(@Loc[nameof(ControlsStrings.ResourceDropdownReplicaAccessibleTitle)], ViewModel.Name, ViewModel.Id.ReplicaSetName)">@ViewModel.Name</span>
```

Comment:
I noticed the div+width was compressed when the display name is long. padding is always applied.